lib/shopify_api/model/webhook_scope.ex

```diff
@@ -0,0 +1,22 @@
+defmodule ShopifyAPI.Model.WebhookScope do
```

Comment:
Does Shopify use the term `scope` to refer to webhooks anywhere? Just curious how you landed on this struct name.

All I could find in Shopify's docs is that the headers contain "webhook metadata". https://shopify.dev/docs/apps/build/webhooks#headers

Reply:
Ah, nice, makes sense!